presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/jmx/JdbcClientStats.java

```diff
@@ -37,13 +37,16 @@
     private final JdbcApiStats getPreparedStatement = new JdbcApiStats();
     private final JdbcApiStats getSchemaNames = new JdbcApiStats();
     private final JdbcApiStats getSplits = new JdbcApiStats();
+    private final JdbcApiStats getSystemTable = new JdbcApiStats();
     private final JdbcApiStats getTableHandle = new JdbcApiStats();
     private final JdbcApiStats getTableNames = new JdbcApiStats();
     private final JdbcApiStats getTableStatistics = new JdbcApiStats();
+    private final JdbcApiStats isLimitGuaranteed = new JdbcApiStats();
     private final JdbcApiStats renameColumn = new JdbcApiStats();
     private final JdbcApiStats renameTable = new JdbcApiStats();
     private final JdbcApiStats rollbackCreateTable = new JdbcApiStats();
     private final JdbcApiStats schemaExists = new JdbcApiStats();
+    private final JdbcApiStats supportsLimit = new JdbcApiStats();
```

Comment:
I haven't added them in purpose. Such methods do not need to be measured.

Reply:
Makes sense. We can implement them as a pure delegate so `assertAllMethodsOverridden` passes (and we know if we miss any future methods).

Should we keep it for `getSystemTable`? That could potentially read table metadata. (though that would need to do it internally and bypass any cache...)